client/plasma_cash/interface.go

```diff
@@ -39,7 +40,7 @@ type ChainServiceClient interface {
 	SubmitBlock() error
 	Deposit(deposit *pctypes.DepositRequest) error
 
-	SendTransaction(slot uint64, prevBlock *big.Int, denomination *big.Int, newOwner string, prevOwner string, sig []byte) error
+	SendTransaction(slot uint64, prevBlock *big.Int, denomination *big.Int, newOwner string, prevOwner string, sig []byte, hash []byte) error
```

Comment:
Don't think we need this hash anymore, we have all the fields we need to reconstruct it from the PlasmaTx in the Go contract.